src/components/ApiHistoryTable.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { Details } from '@docusaurus/theme-common/Details';
+import {
+  ApiHistory,
+  PrReleaseVersions,
+  PrReleaseVersionsContainer,
+} from '../transformers/api-history';
+import ReactMarkdown from 'react-markdown';
+
+enum Change {
+  ADDED = 'API ADDED',
+  CHANGED = 'API CHANGED',
+  DEPRECATED = 'API DEPRECATED',
+}
+
+interface ApiHistoryTableProps {
+  apiHistoryJson: string;
+  prReleaseVersionsJson: string;
+}
+
+// TODO: Add styling based on type
+function generateTableRow(
+  prReleaseVersions: PrReleaseVersionsContainer,
+  type: Change,
+  prUrl: string,
+  changes?: string
+) {
+  const prNumber = prUrl.split('/').at(-1);
+
+  const releaseStatus = prReleaseVersions[Number(prNumber)];
+  // TODO: Sorting
```

Comment:
You can make use of `compareVersions` from `@electron/fiddle-core` here.

Reply:
The table will only include released major versions so I'm not sure if that's necessary. Can just do:

```js
[ "30.0.0", "29.0.0", "31.0.0" ].toSorted((a, b) => b.split(".")[0] - a.split(".")[0])
```